libs/api/domains/health-insurance-v2/src/lib/graphql/models/accidentNotificationConfirmation.model.ts

```diff
@@ -0,0 +1,18 @@
+import { Field, ObjectType } from '@nestjs/graphql'
+import { IsEnum } from 'class-validator'
+
+export enum ConfirmationTypes {
+  INJUREDORREPRESENTATIVEPARTY = 'InjuredOrRepresentativeParty',
+  COMPANYPARTY = 'CompanyParty',
+}
+
+@ObjectType()
+export class AccidentNotificationConfirmation {
```

Comment:
Should follow the [graphql naming convention](https://docs.devland.is/technical-overview/api-design-guide/graphql-naming-conventions#integrating-naming-conventions-into-shared-api).